### PR TITLE
[frontend] Kill chain phases display in the overview of a STIX Core Relationship (#3141)

### PR DESCRIPTION
**Pull request by Archidoit**
## related issue
https://github.com/OpenCTI-Platform/opencti/issues/3141

Components with Kill Chain Phases in their overview:
- stix core relationship
- tool
- malware
- indicator
- infrastructure
- attack patterns

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.js [hunk 1]
```diff
@@ -420,6 +431,27 @@ class StixCoreRelationshipContainer extends Component {
                           stixCoreRelationship.description
                         )}
                       </Markdown>
+                      <Typography variant="h3" gutterBottom={true} style={{ marginTop: 20 }}>
+                        {t('Kill chain phases')}
+                      </Typography>
+                      <List>
```
Comment (RomuDeuxfois) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.js]: Do you think we can have a generic component like StixCoreObjectLabelsView ?
To manage in the same way the kill chain in Tool, Malware and now StixCoreRelationship.

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.js]: right ! I'll do this !

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.js:437]: right ! I'll do this !

**Comment by RomuDeuxfois** [comment]
Nice work !

**Review by RomuDeuxfois — APPROVED** [review]

**Pull request merged**